slather.gemspec

```diff
@@ -27,6 +27,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "equivalent-xml", "~> 0.5.1"
 
   spec.add_dependency "clamp", "~> 0.6"
-  spec.add_dependency "xcodeproj", "~> 0.23.0"
-  spec.add_dependency "nokogiri", "~> 1.6.3"
+  spec.add_dependency "xcodeproj", "~> 0.23"
```

Comment:
I'm not sure this is a good idea, remember there could be API incompatibilities between minor versions when the API is < 1.0.0 in semantic versioning.

There is no guarantee that Xcodeproj 0.25.0 will work when it's released with Slather.